### PR TITLE
Add Ventus@0.2.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by dakshshah96**
Pull request for issue: #8684 
Related issue(s): #11170 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rlamana/Ventus
 * Official website (optional, not the repository): http://www.rlamana.com/ventus/
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 28
   - Count of stars: 193
   - Count of forks: 57
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/Ventus/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Ventus",
+  "version": "0.2.1",
```
Comment (sufuf3) [inline ajax/libs/Ventus/package.json]: Oh~ in commit message the version is v0.2.2, And here is v0.2.1
Which one is correct?

Comment (dakshshah96) [inline ajax/libs/Ventus/package.json]: The latest version is 0.2.2 according to https://github.com/rlamana/Ventus/releases/latest

I'll update the `package.json`.

**Comment by dakshshah96** [comment]
@sufuf3 Done 👍

**Comment by PeterDaveHello** [comment]
Looks like you put the files under `0.2.1` not `0.2.2`?

**Comment by dakshshah96** [comment]
@PeterDaveHello My bad, fixed 👍 
cc @sufuf3

**Pull request merged**